contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -893,16 +893,16 @@ get_pltsql_function_signature(PG_FUNCTION_ARGS)
 	const char	*func_signature;
 
 	proctup = SearchSysCache1(PROCOID, ObjectIdGetDatum(funcoid));
-	if (!HeapTupleIsValid(proctup))
-		elog(ERROR, "cache lookup failed for function %u", funcoid);
-	form_proctup = (Form_pg_proc) GETSTRUCT(proctup);
-
-	func_signature = (char *) get_pltsql_function_signature_internal(NameStr(form_proctup->proname),
+	if (HeapTupleIsValid(proctup)){
```

Comment:
Is this error case valid for dump/restore or in call cases? Otherwise, it would have better to ignore it only in dump restore context